# Add the option "shared-access" to scdaemon.
# If set, pcsc_connect is called with PCSC_SHARE_SHARED instead of PCSC_SHARE_EXCLUSIVE.
# This is officially supported now by GnuPG since at least 2.2.32, but the option is called differently.
# To keep GPG Suites configuration working as well, add a shared-access option in addition to the pcsc-shared.

diff --git a/scd/scdaemon.c b/scd/scdaemon.c
index 5491556..5ac7dba 100644
--- a/scd/scdaemon.c
+++ b/scd/scdaemon.c
@@ -141,6 +141,7 @@ static ARGPARSE_OPTS opts[] = {
   ARGPARSE_s_s (opcscDriver, "pcsc-driver",
                 N_("|NAME|use NAME as PC/SC driver")),
   ARGPARSE_s_n (opcscShared, "pcsc-shared", "@"),
+  ARGPARSE_s_n (opcscShared, "shared-access", N_("use PCSC_SHARE_SHARED for pcsc_connect")),
   ARGPARSE_s_n (oDisableCCID, "disable-ccid",
 #ifdef HAVE_LIBUSB
                                     N_("do not use the internal CCID driver")
